Documentation/gettingstarted.rst

```diff
@@ -9,6 +9,242 @@ your laptop.  It is designed to take 15-30 minutes.
 
 If you haven't read the :ref:`intro` yet, we'd encourage you to do that first.
 
+Getting Started using Kubernetes
+--------------------------------
+
+This guide is using minikube to demonstrate deployment and operation of Cilium
+in a Kubernetes cluster. If instead you want to dive right into the details of
+deploying Cilium on a full fledged Kubernetes cluster, then go straight to the
+<<k8s install ref>>.
+
+The best way to get help if you get stuck is to ask a question on the `Cilium
+Slack channel <https://cilium.herokuapp.com>`_ .  With Cilium contributors
+across the globe, there is almost always someone available to help.
+
+Step 0: Install minikube & kubectl
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Install ``kubectl`` as described in the `Kubernetes installation guide
+<https://kubernetes.io/docs/tasks/kubectl/install/>`_.
+
+Install ``minikube`` 0.19 as described in the guide `Running Kubernetes Locally
+via Minikube
+<https://kubernetes.io/docs/getting-started-guides/minikube/#installation>`_.
+
+::
+
+    $ minikube start --network-plugin=cni --iso-url https://github.com/cilium/minikube-iso/raw/master/minikube.iso
+
+.. note:: The ``--iso-url`` is required to run a recent enough kernel. The base
+          ISO image of minikube has since been updated in the development
+          branch of minikube. As soon as the minikube 0.19 ISO is released,
+          passing the ``-iso-url`` parameter will no longer be required.
+
+After minikube has finished  setting up your new Kubernetes cluster, you can
+check the status of it by running ``kubectl get cs``:
+
+::
+
+    $ kubectl get cs
+    NAME                 STATUS    MESSAGE              ERROR
+    controller-manager   Healthy   ok
+    scheduler            Healthy   ok
+    etcd-0               Healthy   {"health": "true"}
+
+Step 1: Deploy Cilium
+^^^^^^^^^^^^^^^^^^^^^
+
+The next step is to deploy Cilium to your Kubernetes cluster in the form of a
+DaemonSet_ which will deploy one Cilium pod per cluster node in the
+``kube-system`` namespace along with all other system relevant daemons and
+services.
+
+To deploy Cilium, run ``kubectl create -f`` and pass in the file ``cilium-ds.yaml``
```

Comment:
The issue seems to be that the `localkube` as part of your minikube version is still on `v1alpha1` APIs whereas the RBAC roles in the yaml are:
```
apiVersion: rbac.authorization.k8s.io/v1beta1
```
I think you can just change this to 
```
apiVersion: rbac.authorization.k8s.io/v1alpha1
```

I think we'll have to provide a guide without RBAC or be very clear about the exact version of minikube to run.